pepper-apis/studybuilder-cli/tenants/atcp/actions/google-signin-restriction.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * This Action was migrated from Rule.
+ * Rule name: Google Sign In Restriction
+ * Rule ID: rul_
+ * Created on 9/24/2024
+ */
+
+/**
+ * Handler that will be called during the execution of a PostLogin flow.
+ *
+ * @param {Event} event - Details about the user and the context in which they are logging in.
+ * @param {PostLoginAPI} api - Interface whose methods can be used to change the behavior of the login.
+ */
+exports.onExecutePostLogin = async (event, api) => {
+  /**
+   * The following code block will skip this Action if Rule "Google Sign In Restriction"
+   * was previously executed in the transaction in order to avoid duplication
+   * of logic.
+   */
+  if (api.rules.wasExecuted("change_me")) {
```

Comment:
This code to skip this action execution if rule was executed will be removed post testing.